src/it/test19-intermediate-directories/verify.groovy

```diff
@@ -1,7 +1,7 @@
 def verifyFileInodes() {
-    Process proc = ['rpm', '-q', '--queryformat', '[%{FILEINODES} %{FILEMODES:perms} %-13{FILEUSERNAME} %-14{FILEGROUPNAME} %{FILENAMES}\n]', basedir.path + '/target/*.rpm'].execute() | 'sort -n'.execute()
+    Process proc = ['rpm', '-q', '--queryformat', '[%{FILEINODES} %{FILEMODES:perms} %-13{FILEUSERNAME} %-14{FILEGROUPNAME} %{FILENAMES}\n]', basedir.toString().replace(File.separator, "/") + '/target/test19.rpm'].execute()
     proc.waitFor()
-    return proc.in.getText().trim()
+    return proc.in.getText().split().sort()
```

Comment:
As the sort order seems important here, we should not change this.

Reply:
It should just be a "better" way of keeping the same behavior as before.

I removed the call to `sort -n` (which only works in a shell) and replaced it with pure Groovy `split().sort()`, so

Reply:
Note that this pull request currently causes some test failures, but this one passes.

I think that the test failures are correct now and show some issues with the packager code.

Reply:
Ah sorry, I missed that.